### DISCUSSION TITLE
Clearable props is removed in Select Component in V7

### DISCUSSION THREAD

**Discussion by ghost** [mantinedev/mantine#4847] [Q&A] [answered]
Hello. I am migrating a Dashboard from v6.0.17 to v7.0.0. We have quite a lot of `Select` components. I am unable to find the `clearable` props in the `Select` component. If the `clearable` prop is removed what is the alternative for it.
Thanks for your response

**Comment by zhulien-ivanov** [comment, marked as answer]
Use an `ActionIcon` and `Select`'s `rightSection` property to recreate the button yourself. You can see it implemented in `Combobox` here: https://mantine.dev/combobox/?e=SelectClearable.

**Comment by ghost** [comment]
Thanks, Brother

**Comment by rtivital** [comment]
Clearable prop was added back to these components, it will be available in 7.0.1

**Reply by ghost** [reply]
Thanks, Brother. We used `clearable` a lot.
